.github/Dockerfile.distroless

```diff
@@ -0,0 +1,7 @@
+FROM gcr.io/distroless/static-debian12
```

Comment:
Nit: It's kind of annoying that we don't get syntax highlighting here b/c we don't have a `.Dockerfile` postfix. I'm assuming you chose this name ordering for a reason though. So if that is the case then 🤷 I guess we just deal with no syntax highlighting.

Reply:
No specific reason for ordering; changed to `distroless.Dockerfile` and it looks like syntax highlighting is back!